kratos/processes/replace_elements_and_condition_process.cpp

```diff
@@ -20,22 +20,126 @@
 // Project includes
 #include "processes/replace_elements_and_condition_process.h"
 #include "utilities/parallel_utilities.h"
+#include "utilities/string_utilities.h"
 
 namespace Kratos {
 namespace {
 
-/// Replace entities in a given container if the entity id is present in a list of ids.
-    /*  @param rReferenceEntity New type of entity that will replace old one
-     *  @param rEntityContainer Container of elements susceptible to be replaces
-     *  @param rSetOfIds Set of entities ids we want to replace
-     */
-template <class TEntity, class TEntityContainer>
+std::string GetElementName(const GeometryData::KratosGeometryType ElementType)
```

Comment:
This is quite handy, can you please move this somewhere else? (Preferably separate PR for fat approval 😃)

We should have some utilities file where this would fit no?

Reply:
If so, I'd call it `GetGeometryName`.